src/containers/products/ChronicleProductPage.js

```diff
@@ -5,30 +5,148 @@
-    margin: 300px auto 400px;
+    top: 643px;
   }
 `;
 
 const ChronicleProductPage = () => (
   <>
     <ScrollToTopOnMount />
-    <PageSection bgColor={NEUTRALS.WHITE} includeMenu>
-      <ComingSoonContent>
-        <PageIntro>Coming Soon</PageIntro>
-      </ComingSoonContent>
+    <PageSection bgColor={NEUTRALS.GRAY_05} includeMenu>
+    <IntroContent>
+    <ProductStamp>
+    <Icon src={ChronicleIcon} />
+    <div>CHRONICLE</div>
+    </ProductStamp>
+    <ChronicleIntro>
+    A novel, passive sensing research tool for Android 
+      that collects mobile device usage data with low participant burden.
+    </ChronicleIntro>
+    <ChronicleDescription>
+      {`With continuous time stamped output about app usage, research teams can 
+        calculate detailed variables that reflect the complex ways that children 
+        and adults use mobile media.`}
+    </ChronicleDescription>
+    </IntroContent>
+    <ChronicleProductShot bgImage={ProductImg} />
     </PageSection>
-    <FooterSection />
+    <PageSection>
+      <ChronicleExtendedFeaturesContent>
+        <ProductOverviewWrapper>
+          <ProductOverviewDescriptionWrapper>
+            <ExtendedFeatureHeader>Easy to Enroll and Monitor Participants</ExtendedFeatureHeader>
+            <ExtendedFeatureDescription>
+              {`Participants only need to download the application onto their 
+                smartphone or tablet and sign in with the study ID and link you 
+                provide them. Participants only need to download the Chronicle 
+                application from the Google Play Store and click a specialized 
+                link sent to them from the study team which will input their 
+                personalized login information into the app to enroll their device. 
+                The Chronicle application will upload data to the OpenLattice 
+                platform every 15 minutes while the device is connected to WiFi. 
+                If participants wish to stop their data collection at any time, 
+                all they need to do is uninstall the app.. `}
+            </ExtendedFeatureDescription>
+          </ProductOverviewDescriptionWrapper>
+          <FeatureShot bgImage={FeatureImg1} />
+        </ProductOverviewWrapper>
+        <ProductOverviewWrapper>
+          <FeatureShot bgImage={FeatureImg2} />
+          <ProductOverviewDescriptionWrapper>
+            <ExtendedFeatureHeader>How it Works</ExtendedFeatureHeader>
+            <ExtendedFeatureDescription>
+              {`Chronicle objectively measures the duration, content, and timing 
+                of Android mobile device use. Researchers receive an output which 
+                includes every user interaction, application launch and termination, 
+                how often the device is checked, and other data points all with 
+                appropriate time stamps. Data are stored on HIPAA-compliant servers 
```

Comment:
Shouldn't it be `Data is stored...` instead of `Data are stored...`?

Reply:
Technically it is plural (one datum, multiple data), and any academic will gladly point that out, lol.
UM wrote this, and since they're the target group, I'd leave it as is :)

Reply:
I strongly prefer "Data is" over "Data are", but I won't block the PR over this; it's an easy change if we decide to change it.